### DISCUSSION TITLE
Display IP after conection

### DISCUSSION THREAD

**Discussion by DalmaFernandez** [khoih-prog/ESP_WiFiManager#100] [Q&A]
Hi! What example should I use to display my IP address after a successful connection? On the other hand, I would also like to serve my application on a web server. I use ESP32s nodemcu

**Comment by khoih-prog** [comment]
Any example

> [WM] DNS1 and DNS2 set
> [WM] setWifiStaticIP IP = 192.168.2.186
> [WM] Connected after waiting (s) : 0.12
> [WM] Local ip = 192.168.2.186          <==================
> [WM] Timed out connection result: WL_CONNECTED
> WiFi connected...yeey :)
